src/gluonts/model/statsforecast/predictor.py

```diff
@@ -0,0 +1,301 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import List, Optional
+
+import numpy as np
+
+from statsforecast.models import (
+    ADIDA,
+    AutoARIMA,
+    AutoCES,
+    CrostonClassic,
+    CrostonOptimized,
+    CrostonSBA,
+    ETS,
+    IMAPA,
+    TSB,
+)
+
+from gluonts.core.component import validated
+from gluonts.dataset import DataEntry
+from gluonts.model.predictor import RepresentablePredictor
+from gluonts.model.forecast import QuantileForecast
+
+
+# currently it's tricky to get quantiles out of statsforecast
+# see https://github.com/Nixtla/statsforecast/issues/256
+# therefore we need the following function
+def quantiles_to_intervals(quantile_levels: Optional[List[float]] = None):
+    if quantile_levels is None:
+        return None, dict()
+
+    intervals = set()
+    keys = dict()
+
+    for ql in quantile_levels:
```

Comment:
I don't like abbrevations!
```suggestion
    for quantile_level in quantile_levels:
```